bin/dogescript.js

```diff
@@ -9,6 +9,9 @@ var argv     = require('optimist').usage('Usage: dogescript <file>').argv;
 var beautify = require('js-beautify').js_beautify;
 var parser   = require('../lib/parser');
 
+// display version message, keep pre for dev
+process.stdout.write("[dogescript@2.4.0-pre]\n");
```

Comment:
Can we read this in from a VERSION string (maybe read it in from the package json or a VERSION.txt file)  somewhere? Would like to keep all of our versions in a single place so we can bump them easily

Reply:
yeah i thought of that part too :)